code/game/objects/items/weapons/swords_axes_etc.dm

```diff
@@ -289,3 +289,49 @@
 	name = "hardwood wakibokken blade"
 	desc = "A sturdy wooden wakizashi blade, used for training on old Terrae."
 	icon_state = "wakibokken_blade_h"
+
+/obj/item/bo_staff
+	name = "stave"
+	desc = "A thick rod of hardened wood, useful as a walking stick, as much as a defensive tool."
+	icon = 'icons/obj/weapons.dmi'
+	icon_state = "wakibokken_blade_h"
+	force = 15
+	slot_flags = SLOT_BACK
+	sharp = 1
+	hitsound = "swing_hit"
+	attack_verb = list("smashed", "slammed", "whacked", "thwacked")
+	icon_state = "bostaff0"
+	item_state = "bostaff0"
+	var/defend_chance = 40
+	var/projectile_parry_chance = 0
+
+/obj/item/bo_staff/proc/jedi_spin(mob/living/user)
+	for(var/i in list(NORTH,SOUTH,EAST,WEST,EAST,SOUTH,NORTH,SOUTH,EAST,WEST,EAST,SOUTH))
+		user.setDir(i)
+		if(i == WEST)
+			user.emote("flip")
+		sleep(1)
+
+/obj/item/bo_staff/attack(mob/target, mob/living/user)
+	add_fingerprint(user)
+	if(!issilicon(target))
+		return ..()
+	if(!isliving(target))
+		return ..()
+	if(user.a_intent == INTENT_DISARM)
+		if(!ishuman(target))
```

Comment:
since you're checking human anyways, this entire segment should use
```
if(!ishuman(target))
    return
if(user.a_intent != INTENT_DISARM)
    return
// do your stuff here
```